Xamarin.PropertyEditing.Mac/Controls/PointEditorControl.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Drawing;
+using AppKit;
+using CoreGraphics;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class PointEditorControl : PropertyEditorControl
+	{
+		internal NSTextField XEditor { get; set; }
```

Comment:
Generally I don't like doing commits to fix formatting or stuff like this because it makes the history harder to get useful information from. I prefer to squash it into the original commit before it goes in, or to fix it when there's another change touching that line, at least ideally.